modules/mono/editor/godotsharp_builds.h

```diff
@@ -92,7 +92,7 @@ class GodotSharpBuilds {
 
 	static bool make_api_assembly(APIAssembly::Type p_api_type);
 
-	static bool build_project_blocking(const String &p_config);
+	static bool build_project_blocking(const String &p_config, const Vector<String> &p_defines);
```

Comment:
Can we rename this to `p_godot_defines` both here and in the source file? This is to avoid confusion as we prepend the `GODOT_` suffix.